app/rww/ldp/WebClient.scala

```diff
@@ -236,9 +235,7 @@ class WSClient[Rdf <: RDF](readerSelector: ReaderSelector[Rdf, Try], rdfWriter:
     }
   }
 
-  override
-  def put[S](url: Rdf#URI, graph: Rdf#Graph, syntax: Syntax[S])
-            (implicit writer: Writer[Rdf#Graph, Try, S]): Future[Unit] = {
+  override def put[S](url: Rdf#URI, graph: Rdf#Graph, syntax: Syntax[S])(implicit writer: Writer[Rdf#Graph, Try, S]): Future[Unit] = {
```

Comment:
putting all this on  one line does not make the code clearer. My guess is that your version IDE does this perhaps. It looks like a lot of the changes are done automatically by your IDE.